build/server-image/kube-apiserver/Dockerfile

```diff
@@ -13,17 +13,18 @@
 # limitations under the License.
 
 # This file create the kube-apiserver image.
-ARG BASE_IMAGE_REGISTRY
-ARG BASE_IMAGE_NAME
+ARG BASE_IMAGE
```

Comment:
suggest using BASEIMAGE as it was prior to https://github.com/kubernetes/kubernetes/pull/99379/files to make release-over-release diffs on these files highlight actual changes

Reply:
also done